core/templates/css/oppia.css

```diff
@@ -2184,10 +2184,14 @@ pre.oppia-pre-wrapped-text {
   outline: 2px solid #009688;
 }
 .oppia-interaction-tile-name {
+  align-items: center;
```

Comment:
Apart from height, for all changes made, there is already a class for this. Can you use that instead? Use `fx-row fx-main-center fx-cross-center`. It will make it more specific.

Reply:
@srijanreddy98 Done